website/docs/addon_maya_ornatrix_artist.md

```diff
@@ -0,0 +1,101 @@
+
+![Maya - Ornatrix Rig Publishing](assets/maya/artist/ornatrix_rig_publishing.gif)
+
+Select your mesh (that has with Ornatrix fur) and create the Ornatrix Rig instance.If the instance contains anything other than the mesh, the validator checks and raise error to make sure there are always mesh(s) inside the instance. When it is published, several files are exported to the publish folder:
+
+- Maya Scene with the Ornatrix data (.ma)
+- Groom Preset (.oxg.zip)
+- Rig Setting data (.rigsettings)
+- Ornatrix Almebic (.abc)
+- Cache Setting data (.cachesettings)
+
+### Loading
+
+![Maya - Ornatrix Loader Product Type](assets/maya/artist/ox_loader_product_type.png)
+
+**Ornatrix Rig** supports to load maya scene (.ma), groom preset (.oxg.zip), Ornatrix alembic (.abc) and Ornatrix alembic to Hair Guide Node (.abc).
+
+![Maya - Ornatrix Loaders](assets/maya/artist/ox_rig_loader.png)
+
+Both Maya Scene and groom preset allow for connecting ornatrix rig with the animation/pointcache rig.
+
+![Maya - Ornatrix Rig Loading](assets/maya/artist/ornatrix_rig_loading.gif)
+
+#### Connecting an Ornatrix rig with a pointcache or animation
+After loading the Ornatrix Rig and a pointcache to connect it to, you can use the AYON tools to connect it for you:
+
+1. Go to `AYON` -> `Manage...`
+2. In the inventory select the assets from oxrig and animation/pointcache product type.
+3. Right-click `Actions` -> `Connect Ornatrix Rig`
+The Ornatrix geometry will be connected the animation rig with the transform and mesh attributes.
+
+:::tip
+The connecting will occur by node name - so if it fails, make sure the node name for the geometry is the same across your animation caches and the ornatrix exports.
+**Double geometry:** Be aware that both the ornatrix rig and your animated cache have the mesh geometry. After connecting the fur you will most likely want to hide one of the two to ensure you do not have two overlapping meshes during rendering.
+:::
+
+#### Loading Ornatrix Hair Guide using the Ornatrix alembic
+
+Users can choose to load Hair Guide Node with Alembic by Ornatrix alembic to Hair Guide Node (.abc), which sets the alembic as file cache path inside the node and they can select to connect the node with the node editor or edit it in the operator stack editor.
+
+Once you are finished with the Ornatrix rig, you can go to `AYON` -> `Publish` to publish an Ornatrix Cache.
+
+:::tip
+Loading the Ornatrix Rig will automatically create the Ornatrix Cache instance for it. That's why you can directly continue publishing data when you're ready.
+
+You can turn off the auto-creation by going to `AYON` server -> Project settings > `Maya` > `Loaders` > `Ornatrix Rig Loader` -> `Create Ornatrix Cache on load`
+:::
+
+
+## Ornatrix Caches
+
+### Creating and publishing
+
+Users can either create an ```Ornatrix Cache``` instance themselves or load ornatrix rig which automatically creates the instance ready for publishing.
```

Comment:
```suggestion
Create an ```Ornatrix Cache``` instance or load an ornatrix rig which automatically creates the instance ready for publishing.
```

---

Question: The ornatrix cache and rigs only ever support ONE mesh with ONE ornatrix node, right? Or can it export multiples into one ornatrix product?

And... can you add a paragraph here @moonyuet which describes what you need to put INTO the Ornatrix Cache instance. Is it the mesh? Is it the Ornatrix node? Is it just one mesh or multiple?